test/DynamoCoreWpfTests/CodeBlockNodeTests.cs

```diff
@@ -31,7 +32,7 @@ protected override void GetLibrariesToPreload(List<string> libraries)
         //Execute(InitialRun)
         //Update the value from 10 to 0
         //Execute(SecondRun)
-        [Test, RequiresSTA]
+        [Test, Apartment(ApartmentState.STA)]
```

Comment:
The apartment state attribute can be set on the test, class or assembly
I wonder if it would be simpler to just set it in the AssemblyInfo.cs for all these test projects...
https://docs.nunit.org/articles/nunit/writing-tests/attributes/apartment.html

Reply:
Won't this mean that the attribute gets applied to all tests in that assembly?

Reply:
yes, I think.
Not sure where we need MTA vs STA or why...